server/resource_server/emails/verification/verifier_test.go

```diff
@@ -55,7 +55,7 @@ func TestGenerateVerificationCode(t *testing.T) {
 		now,
 	)
 
-	generatedCode := verifier.GenerateVerificationCode(
+	generatedCode, _ := verifier.GenerateVerificationCode(
```

Comment:
Can we not check the error here also? Instead of using _

Reply:
Added the error check